client_push_go17.go

```diff
@@ -0,0 +1,50 @@
+// +build go1.7
```

Comment:
I think it better use !go1.6 here for >=1.8 compatibility. Anyway go < 1.6 is not supported by this lib